src/servicesHost.ts

```diff
@@ -27,8 +27,17 @@ function makeServicesHost(
     const resolveSync = makeResolver(loader.options);
 
     const moduleResolutionHost = {
-        fileExists: (fileName: string) => utils.readFile(fileName) !== undefined,
-        readFile: (fileName: string) => utils.readFile(fileName),
+        fileExists: (fileName: string) => {
```

Comment:
Cool - could we document that in comments please?  Perhaps it would be worth this code path only being executed if `usePreviousLoaderGeneratedFiles` is true; presumably otherwise it would not work if there weren't previous loaders in the chain?